### DISCUSSION TITLE
Unusual . operator error while accessing table of a database

### DISCUSSION THREAD

**Discussion by Vedant817** [prisma/prisma#23642] [Q&A]
### Bug description
```js
const products = await db.product.findMany({
        select: {
            id: true, name: true, priceInCents: true, isAvailableForPurchase: true, _count: {
                select: { orders: true }
            }
        },
        orderBy: { name: "asc" }
    })
 ```  
   Error while access the product table of the db database while it is correct using nextjs

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @Vedant817 👋 

Thank you for raising this question.

It seems you are getting an error when trying to access the `product` table of your database using Prisma Client. There are several reasons why this might be the case:


1. The `product` table might not exist in your database. Ensure that the product table exists in your database and matches the `product` model in your Prisma schema.

2. Your Prisma Client might not be up-to-date with your Prisma schema. If you've recently made changes to your Prisma schema, you need to regenerate your Prisma Client by running `npx prisma generate`.

3. There might be a synchronization issue between your Prisma schema and your database schema. You can use `prisma db push` or `prisma-migrate` to sync your Prisma schema with your database schema.

If none of the above works for you, kindly share the error message to enable us debug this further.

**Comment by 0xKaso** [comment]
i have the same problem, prmisa not work but everything is ok

and no error has been catch

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
